src/makeChange.js

```diff
@@ -0,0 +1,28 @@
+
+export default function makeChange({price, amountGiven}) {
+  let coins = {
+    quarters: 0,
+    dimes: 0,
+    nickels: 0,
+    pennies: 0
+  }
+  const change = (amountGiven - price)
+  if(change < 0){
+    return 'No change provided'
+  }
+  const quarters = Math.floor(change / 25)
```

Comment:
Think about refactoring this logic to reduce lines of code and to adhere to the DRY principle.

The only parts of this code that are changing is the integer that is being used as the divisor and multiplicand for each denomination, as well as the string that is being used to access the `coins` object.